packages/react/src/components/lastUpdated/LastUpdatedConnected.tsx

```diff
@@ -22,4 +22,7 @@ const mapDispatchToProps = (
     onDestroy: () => dispatch(removeLastUpdated(ownProps.id)),
 });
 
+/**
+ * @deprecated use LastUpdated instead
+ */
 export const LastUpdatedConnected = connect(mapStateToProps, mapDispatchToProps, ReduxUtils.mergeProps)(LastUpdated);

```

Comment:
I just looked in the Admin project we don't use it anymore.

Reply:
Yep! That's why I marked it as deprecated.

We'll do one pass soon to remove a bunch of deprecated components in one go